bin/amphtml-update.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
-set -e
+# Update includes/sanitizers/class-amp-allowed-tags-generated.php based on the AMPHTML validator spec.
+#
+# To update to the latest release of AMPHTML:
+#
+# $ ./amphtml-update.sh
+#
+# Update to a specific commit of AMPHTML
+#
+# $ git clone git@github.com:ampproject/amphtml.git amphtml
+# $ cd amphtml; git checkout ec5fd60; cd -
+# $ ./amphtml-update.sh amphtml/
 
-# Go to the right location.
-cd "$(dirname "$0")"
+set -e
 
-BIN_PATH="$(pwd)"
-PROJECT_PATH=$(dirname $PWD)
-VENDOR_PATH="$PROJECT_PATH/vendor"
+BIN_PATH="$(dirname "$0")"
+PROJECT_PATH=$(dirname $BIN_PATH)
+AMPHTML_LOCATION="$1"
 
 if ! command -v apt-get >/dev/null 2>&1; then
```

Comment:
While we're at it, can we move this check to within the Python checks? As far as I can see, one doesn't need `apt-get` when Python is already available.

Reply:
I don't think so. It's not whether `python` isn't available, but rather whether Python _and_ the protobuf packages are available.

Reply:
Rephrase: let‘s move these checks to after the python and protobuf checks then.

In other words: this would prevent me from running the script on my Mac.

Reply:
I'm not sure I understand. The script is supposed to prevent you from running on your Mac since apt is required.

Reply:
Looking at the code, I see that python is required, not apt specifically. Is that incorrect?

Reply:
These packages are required: `python`, `protobuf-compiler`, `python-protobuf`.

We could check to see if those Python modules are installed directly instead instead of doing so via apt, but we can do that in another PR.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/1891/commits/6ae9abb7cd6948a85dc896cb35207cc457c5516f